docker-compose.yml

```diff
@@ -1,32 +1,38 @@
 version: '3.7'
-services:
 
-  webserver:
-    image: nginx:alpine
-    container_name: badge-poser-webserver
-    working_dir: /application
-    depends_on:
-      - php-fpm
-    volumes:
-      - .:/application:cached
-      - ./docker/dev/nginx/nginx.conf:/etc/nginx/conf.d/default.conf:ro
-      - ./var/log/nginx/:/var/log/nginx:cached
+services:
```

Comment:
do you want to use this file for development or production?

Reply:
I guess on dev, as on prod it would be deployed on lambda so no docker orchestrator should be needed